cookiecutter.yaml

```
@@ -11,3 +11,5 @@ default_context:
     hosting_github_url: "https://github.com/OpenMontana/montana-legislature-council-data-project"
     hosting_web_app_address: "https://OpenMontana.github.io/montana-legislature-council-data-project"
     firestore_region: "us-west1"
+    event_gather_timedelta_lookback_days: 2
```

```suggestion
    event_gather_timedelta_lookback_days: 10
```

@whargrove for your tracking. smai had updated this in the app prior to the base app actually tracking it in the default build: https://github.com/OpenMontana/montana-legislature-council-data-project/blob/main/python/event-gather-config.json#L7